src/components/molecules/ArtifactsList/ArtifactsList.tsx

```diff
@@ -0,0 +1,46 @@
+import {useMemo} from 'react';
+
+import {FileOutlined} from '@ant-design/icons';
+
+import {Artifact} from '@models/artifact';
+
+import {downloadFileName} from '@services/artifacts';
+
+import {
+  StyledArtifactsFileName,
+  StyledArtifactsListContainer,
+  StyledArtifactsListItem,
+  StyledDownloadIcon,
+  StyledSpace,
+} from './ArtifactsList.styled';
+
+type ArtifactsListProps = {
+  artifacts: Array<Artifact>;
+  testExecutionId: string;
+};
+
+const ArtifactsList: React.FC<ArtifactsListProps> = props => {
+  const {artifacts, testExecutionId} = props;
+
+  const renderedArtifactsList = useMemo(() => {
+    return artifacts.map((artifact, index) => {
```

Comment:
You can directly return here like `useMemo(() => artifacts.map(...)`, just to save some characters

Reply:
I don't like oneliners 😄